src/Model/Behavior/SequenceBehavior.php

```diff
@@ -363,4 +364,21 @@ protected function _sync($fields, $conditions, $scope = null)
         }
         return $this->_table->updateAll($fields, $conditions);
     }
+
+    /**
+     * Returns the update expression for the order field
+     *
+     * @param string $direction Whether to increment or decrement the field
+     * @param string $amount Amount to modify the field by
+     */
+    protected function _getOrderUpdateExpression($direction = 'increment', $amount = 1) {
```

Comment:
- Method name is a bit too long, just `_getUpdateExpression()` is good enough. 
- No need to use strings `increment`, `decrement` for direction, just directly use `+`, `-`.
- `$amount` param can probably be dropped. I don't think there will be a use case of amount value other than 1.